docs/question.md

```diff
@@ -7,7 +7,7 @@
 
 ## Directory structure
 
-Questions are all stored inside the main `questions` directory for a course. Each question is a single directory that contains all the files for that question. The name of the question directory is the question ID label (the `id`) for that question. For example, here are two different questions:
+Questions are all stored inside the `questions` directory (or any subfolder) for a course. Each question is a single directory that contains all the files for that question. The name of the full question directory relative to `questions` is the question ID label (the `id`) for that question. For example, here are three different questions:
```

Comment:
Should we add a similar comment to the docs about assessments and course instances?